lib/config/presets/internal/default.ts

```diff
@@ -175,7 +175,7 @@ export const presets: Record<string, Preset> = {
   },
   disableLockFiles: {
```

Comment:
@rarkins should we mark this as deprecated and add a new preset or also migrate it to a new name 🤔

Reply:
Migrate it to the new name